auth/src/config/index.ts

```diff
@@ -10,6 +10,6 @@ export namespace Config {
     }
 
     export function isDevMode(): boolean {
-        return process.env.OS4_DEVELOPMENT === '1';
+        return process.env.OPENSLIDES_DEVELOPMENT === '1';
```

Comment:
@GabrielInTheWorld please be a bit more general here. Check if the lower cased variable is in ('1', 'y', 'yes', 'true', 'on').